app/models/atmosphere/virtual_machine_flavor.rb

```diff
@@ -60,6 +66,33 @@ class VirtualMachineFlavor < ActiveRecord::Base
 
     scope :active, -> { where(active: true) }
 
+    # Assumes only 1 vmf_osf for a specific os_family
+    def get_hourly_cost_for(os_family)
+      incarnation = flavor_os_families.find_by(os_family: os_family)
+      incarnation && incarnation.hourly_cost # nil if incarnation.blank?
```

Comment:
I is very simple checkable, just run this in pry or irb:

```ruby
a = nil
a && a[:a]

a = { a: 'a' }
a && a[:a
```

As you can see everything is as expected here :smile: 

```ruby
[18] pry(main)> a = nil
=> nil
[19] pry(main)> a && a[:a]
=> nil
[20] pry(main)> a = { a: 'a' }
=> {:a=>"a"}
[21] pry(main)> a && a[:a]
=> "a"
```